dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/internal/OkHttpUploader.kt

```diff
@@ -66,7 +67,12 @@ internal class OkHttpUploader : Uploader {
         repositoryInfo: RepositoryInfo?
     ): MultipartBody {
         val mappingFileBody = MultipartBody.create(MEDIA_TYPE_TXT, mappingFile)
-
+        if (mappingFileBody.sizeInMb() > MAX_MAP_FILE_SIZE_IN_MB) {
```

Comment:
maybe weird question, but what if file size is 50 MB and 1 byte? it is also not clear from the task if it is inclusive or exclusive bound. maybe we can check that in backend code?

Reply:
check the unit tests...there's a specific test there for that.